templates/problem/search-form.html

```diff
@@ -52,7 +52,7 @@ <h3>{{ _('Problem search') }} <i class="fa fa-search"></i>
                     <select id="types" name="type" multiple>
                         {% for type in problem_types %}
                             <option value="{{ type.id }}"{% if type.id in selected_types %} selected{% endif %}>
-                                {{ type.full_name }}
+                                {{ user_trans(type.full_name) }}
```

Comment:
Would it make sense to add a similar change for problem categories?

Reply:
Problem categories are out of scope. In particular, they require these steps:

- Modify `makedmojmessages` to get all problem categories. Example: https://github.com/DMOJ/online-judge/blob/8eed5e5278f68c35a0b46bcce17c1a64fae5dd78/judge/management/commands/makedmojmessages.py#L127-L130
- Wait for Crowdin translators to translate those strings, and an admin to get those translations.
- Use `user_trans` in the appropriate templates.

Reply:
I see, this is out of scope.